src/renderer/i18n/locales/en-US.js

```diff
@@ -667,6 +667,7 @@ export default {
       REMOVING: 'Removing \'{plugin}\'',
       UPDATING: 'Updating \'{plugin}\'',
       OFFICIAL: 'Official {author} plugin',
+      GRANTS: 'Funded by {author} grants',
```

Comment:
Can this be changed to `Funded by ARK Grants` please?

Reply:
done @avfcpieface :+1:

Reply:
Thank you @brenopolanski